project/AppDependencies.scala

```diff
@@ -4,30 +4,31 @@ import sbt._
 object AppDependencies {
 
   private val catsVersion         = "2.9.0"
-  private val boostrapPlayVersion = "7.12.0"
+  private val boostrapPlayVersion = "8.0.0"
+  private val pekkoVersion        = "1.0.1"
 
   val compile = Seq(
-    "uk.gov.hmrc"               %% "bootstrap-backend-play-28"          % boostrapPlayVersion,
-    "org.typelevel"             %% "cats-core"                          % catsVersion,
-    "com.networknt"              % "json-schema-validator"              % "1.0.70" exclude ("org.slf4j", "slf4j-api"),
-    "com.fasterxml.jackson.core" % "jackson-databind"                   % "2.12.6",
-    "com.lightbend.akka"        %% "akka-stream-alpakka-xml"            % "3.0.4",
-    "com.lightbend.akka"        %% "akka-stream-alpakka-json-streaming" % "3.0.4"
+    "uk.gov.hmrc"               %% "bootstrap-backend-play-30"       % boostrapPlayVersion,
+    "org.typelevel"             %% "cats-core"                       % catsVersion,
+    "com.networknt"              % "json-schema-validator"           % "1.0.70" exclude ("org.slf4j", "slf4j-api"),
+    "com.fasterxml.jackson.core" % "jackson-databind"                % "2.12.6",
+    "org.apache.pekko"          %% "pekko-stream"                    % pekkoVersion,
```

Comment:
is this really required? if not please remove.

Reply:
This part ```"org.apache.pekko"          %% "pekko-stream"                    % pekkoVersion,```?

Reply:
Good shout, removed :)